libs/components/src/lib/select/select.spec.ts

```diff
@@ -537,4 +538,34 @@ describe('vwc-select', () => {
 			expect(element.selectedOptions[0]).toEqual(element.querySelector('option:nth-child(2)'));
 		});
 	});
+
+	describe('placeholder', function () {
+		it('should set selectedindex -1 when placeholder', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} placeholder="placeholder">
+					<option value="1">1</option>
+					<option value="2">2</option>
+					<option value="3">3</option>
+				</${COMPONENT_TAG}>`
+			)) as Select;
+
+			await elementUpdated(element);
+			expect(element.selectedIndex).toEqual(-1);
+		});
+
+		it('should change selectedindex -1 when option selected', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG}>
+					<option value="1">1</option>
+					<option value="2" selected>2</option>
+					<option value="3">3</option>
+				</${COMPONENT_TAG}>`
+			)) as Select;
+
+			element.placeholder = 'placeholder';
+			await elementUpdated(element);
```

Comment:
Why does this test not have the placeholder here?

It doesn't seem to work when setting both the placeholder and `selected` / `value`

```suggestion
		it('should change selectedindex -1 when option selected', async () => {
			element = (await fixture(
				`<${COMPONENT_TAG} placeholder="placeholder">
					<option value="1">1</option>
					<option value="2" selected>2</option>
					<option value="3">3</option>
				</${COMPONENT_TAG}>`
			)) as Select;

			await elementUpdated(element);
```